pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -44,10 +44,10 @@ import (
 )
 
 const (
-	// LabelValue is the value of a label used for the identification of vpn-shoot pods.
-	LabelValue = "vpn-shoot"
-	// ManagedResourceName is the name of the ManagedResource containing the resource specifications.
-	ManagedResourceName = "shoot-core-vpn-shoot"
+	// labelValue is the value of a label used for the identification of vpn-shoot pods.
+	labelValue = "vpn-shoot"
+	// managedResourceName is the name of the ManagedResource containing the resource specifications.
+	managedResourceName = "shoot-core-vpn-shoot"
```

Comment:
This must be exposed since it's used in the health check (`pkg/operation/care/health.go`)

Reply:
yes. just noticed it, the `make-test` was failing.